l10n/en/products/vpn/platforms/mac.ftl

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/mac
+
+# Meta strings
+
+vpn-mac-page-title = { -brand-name-mozilla-vpn } for { -brand-name-mac-short }
+vpn-mac-page-description = Whether on a { -brand-name-mac-short } desktop, laptop, { -brand-name-ipad } or { -brand-name-iphone }, protect your entire device from hackers and prying eyes with a VPN from { -brand-name-mozilla }, a pioneer in internet security since 1998.
+
+# Mac strings
+vpn-mac-hero-headline = Protect your connection online with { -brand-name-mozilla-vpn } for { -brand-name-mac-short }
+
+vpn-mac-ability = From business to leisure, you should feel safe online no matter what brings you there. { -brand-name-mozilla-vpn } helps you use public wifi smarter by protecting your internet connection from hackers and internet providers looking to collect your data. Its one-click privacy is available for { -brand-name-mac-short } and { -brand-name-ios }, (as well as { -brand-name-windows } 10, { -brand-name-linux } and { -brand-name-android }), and doesn’t impact your speed. And, since it’s a VPN from { -brand-name-mozilla } with { -brand-name-wireguard }® protocol, you can trust your activity won’t be logged and your data stays yours — always.
+
+vpn-mac-wifi-headline = Feel safe on sketchy wifi
+vpn-mac-wifi-copy = Go online with peace of mind when you’re using public wifi thanks to { -brand-name-mozilla-vpn }. It protects your internet connection so your personal data is hidden from hackers and prying eyes.
+
+vpn-mac-about-headline = From a trusted tech-maker
+vpn-mac-about-copy = Since 1998, non-profit { -brand-name-mozilla } has been led by its mission to build a better internet. From products like the { -brand-name-firefox-browser } to advocacy work across the globe, { -brand-name-mozilla }’s work continues to ensure the internet is free and open to all.
+
+vpn-mac-privacy-headline = Privacy at your fingertips
+vpn-mac-privacy-copy = With { -brand-name-mozilla-vpn }, your internet connection is encrypted and your IP address is protected from hackers and nosy internet providers thanks to { -brand-name-wireguard }® advanced protocol.
+
+vpn-mac-servers-headline = Access servers around the world
+
+# Variables:
+#   $countries (number) - number of available countries
+#   $servers (number) - number of available servers
+vpn-mac-servers-copy = Set your { -brand-name-mac-short }’s location to your choice of more than { $servers }+ servers in { $countries } countries. This lets you check the news, shop, stream and surf the web from just about anywhere.
```

Comment:
should we add a `+` in in `{ $countries }+ countries` to stay consistent with Linux platform?

Reply:
I think so. There are a few other instances, but that was as approved in the copy doc.